rllib/algorithms/algorithm_config.py

```diff
@@ -1977,6 +1978,19 @@ def evaluation(
                 metrics: dict. See the Algorithm.evaluate() method to see the default
                 implementation. The Algorithm guarantees all eval workers have the
                 latest policy state before this function is called.
+                In case the new `EnvRunner API` is used and
+                `enable_async_evaluation=True` customize the asynchronous evaluation
+                method. This must be a function of signature (algo: Algorithm,
+                eval_workers: WorkerSet, weights_ref: ObjectRef, weights_seq_no: int)
```

Comment:
Ah, I see now.

Ok, sorry, I think we need to fix this in a better way:
Can we try to hide the necessity for the user to handle weights_ref and weights_seq_no themselves? Like what if we only required the user to provide the plain `func(algo, eval_workers)` signature and RLlib auto-wraps this custom function such that it takes care of synching the weights with the eval workers (or NOT) automatically?